### PR TITLE
Us36 

### LINKED ISSUE #36: Merchant Dashboard Invoices sorted by least recent
As a merchant
When I visit my merchant dashboard
In the section for "Items Ready to Ship",
Next to each Item name I see the date that the invoice was created
And I see the date formatted like "Monday, July 18, 2019"
And I see that the list is ordered from oldest to newest

### PR DESCRIPTION
**Pull request by glsorensen**
## Description of Changes

Add test for order of items on page oldest to newest

Add Div to dashboard view

Closes #36

## Concerns (tags recommended)

All good

## What gif describes how you feel about this PR?
frustrated-pulling-hair

### CODE REVIEW

File: spec/features/merchants/dashboard_spec.rb [hunk 1]
```diff
@@ -103,5 +103,14 @@
       expect(page).to have_content(@invoice_4.created_at.strftime("%A, %B %d,%Y"))
       expect(page).to have_content(@invoice_5.created_at.strftime("%A, %B %d,%Y"))
     end
+    it '#US36 lists items oldest to newest' do
```
Comment (gregoryjflaherty) [inline spec/features/merchants/dashboard_spec.rb:106]: Great job testing for order here -  would love to go over the orderly gem more with you if you have time too

File: spec/features/merchants/dashboard_spec.rb [hunk 2]
```diff
@@ -103,5 +103,14 @@
...
+
+      within "div.items_ready_to_ship" do
```
Comment (gregoryjflaherty) [inline spec/features/merchants/dashboard_spec.rb:108]: never seen this syntax before with div. - good to know

File: app/views/merchant_dashboard/index.html.erb [hunk 3]
```diff
@@ -10,9 +10,11 @@
   </section>
 <% end %>
 
-<h4> Items Ready to Ship </h4>
-<% @merchant.items_not_shipped.each do |item| %>
-  <div id="item<%=item.id%>">
-    <li><%= item.name %> - Invoice#<%= link_to item.invoice_id, merchant_invoices_path(@merchant.id, item.invoice_id) %> - <%= item.invoice_created.strftime("%A, %B %d,%Y") %></li>
-  </div>
-<% end %>
+<div class="items_ready_to_ship">
```
Comment (gregoryjflaherty) [inline app/views/merchant_dashboard/index.html.erb:13]: Looks great

File: spec/models/invoice_item_spec.rb [hunk 4]
```diff
@@ -12,4 +12,52 @@
...
+  end
+
+  describe 'instance methods' do
+
+  end
+
+  describe 'class methods' do
+    it '#invoice_created_sorted' do
+      expect(InvoiceItem.invoice_created_sorted(@nike)).to eq([@invoice_item_5, @invoice_item_6])
+      # "#{test.created_at.strftime("%A")}, #{test.created_at.to_s(:long)}"
```
Comment (gregoryjflaherty) [inline spec/models/invoice_item_spec.rb]: is this line meant to be commented out? if so I would remove it

**Review by gregoryjflaherty — APPROVED** [review]

File: spec/models/invoice_item_spec.rb (hunk 4 above)
Comment (glsorensen) [reply spec/models/invoice_item_spec.rb]: Yeah, ill get rid of this, good catch

File: spec/features/merchants/dashboard_spec.rb (hunk 2 above)
Comment (jgrazulis) [reply spec/features/merchants/dashboard_spec.rb:108]: Yea that's interesting-- so is it instead of using the` #`?

**Review by jgrazulis — APPROVED** [review]
Look great!

**Pull request merged**